scss/components/_activity-card.scss

```diff
@@ -150,3 +194,13 @@
   display: block;
   text-align: center;
 }
+
+@keyframes infinite-rotation {
```

Comment:
Keyframes should be a part of a different partial file and should be inside base folder.